addons/account/wizard/accrued_orders.py

```diff
@@ -151,7 +155,7 @@ def _ellipsis(string, size):
                     account = order_line.product_id.property_account_expense_id or order_line.product_id.categ_id.property_account_expense_categ_id
                 else:
                     account = order_line.product_id.property_account_income_id or order_line.product_id.categ_id.property_account_income_categ_id
-                values = _get_aml_vals(order, self.amount, 0, account.id, label=_('Manual entry'))
+                values = _get_aml_vals(order, self.amount, 0, account.id, label=_('Manual entry'), analytic_distribution=order_line.analytic_distribution)
```

Comment:
You have the same issue sorry (with the analytic account on the whole sale order here :smiling_face_with_tear:  )